mirror_builder/bootstrap.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import subprocess
+
+from . import sdist, server
+
+logger = logging.getLogger(__name__)
+
+
+def bootstrap_build_dependencies(ctx):
+    # flit_core is a basic build system dependency for several
+    # packages. It is capable of building its own wheels, so we use the
+    # bootstrapping instructions to do that and put the wheel in the
+    # local server directory for reuse when building other packages via
+    # 'pip wheel'.
+    #
+    # https://flit.pypa.io/en/stable/bootstrap.html
+    sdist_filename = sdist.download_sdist(ctx, ['flit_core'])
+    resolved_name = sdist.get_resolved_name(sdist_filename)
+    sdist_root_dir = sdist.unpack_sdist(ctx, sdist_filename)
+    logger.info('building flit_core wheel in %s', sdist_root_dir)
+    subprocess.check_call(
+        ['python3', '-m', 'flit_core.wheel'],
+        cwd=sdist_root_dir,
```

Comment:
What happened to bootstrap_install.py?

Reply:
That script just unpacks the wheel to install it, but we're already relying on pip to install wheels so we don't need it. It was superfluous in the bash implementation.